libs/sdk-ui-kit/src/Dialog/DialogList/DialogList.tsx

```diff
@@ -32,19 +32,21 @@ export const DialogList: React.VFC<IDialogListProps> = (props) => {
     }
 
     const ListItemComponent = itemComponent ?? DialogListItemBasic;
-    const classNames = cx("gd-dialog-list s-dialog-list", className);
+    const classNames = cx("gd-dialog-list-wrapper s-dialog-list", className);
```

Comment:
Would make more sense have `s` classes named similar way like `s-dialog-list-wrapper`.

Reply:
Wanted to make the test selector more general to the component, but I can change it.